ci/requirements-py37.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - pytest-cov
   - pytest-env
   - coveralls
-  - pycodestyle
+  - flake8
```

Comment:
Can you also replace pycodestyle with flake8 in requirements-py36.yml? Our contributing guide is still pointing users to that for setup.